Makefile

```diff
@@ -81,27 +81,30 @@ hr-docs: ## Generate Docusaurus documentation and blog posts
 SUPERDUPERDB_EXTRAS ?= dev
 
 testenv_image: ## Build a sandbox image
-	@echo "===> Build superduperdb/sandbox"
-	docker build . -f deploy/images/superduperdb/Dockerfile -t superduperdb/sandbox --progress=plain \
+	@echo "===> Build superduperdb/$(SUPERDUPERDB_EXTRAS)"
+	docker build . -f deploy/images/superduperdb/Dockerfile -t superduperdb/$(SUPERDUPERDB_EXTRAS) --progress=plain \
 		--build-arg BUILD_ENV="sandbox" \
 		--build-arg SUPERDUPERDB_EXTRAS="$(SUPERDUPERDB_EXTRAS)"
 
 testenv_init: ## Initialize a local Testing environment
 	@echo "===> Ensure hostnames"
 	@deploy/testenv/validate_hostnames.sh
 
+	@echo "===> Ensure mongodb volume is present"
+	mkdir -p deploy/testenv/$(SUPERDUPERDB_EXTRAS)_data
+
 	@echo "===> Ensure Images"
-	@if docker image ls superduperdb/sandbox | grep -q "latest"; then \
-        echo "superduper/sandbox found";\
+	@if docker image ls superduperdb/$(SUPERDUPERDB_EXTRAS) | grep -q "latest"; then \
+        echo "superduper/$(SUPERDUPERDB_EXTRAS) found";\
         echo "*************************************************************************";\
         echo "** If Dask behaves funny, rebuild the image using 'make testenv_image' **";\
         echo "*************************************************************************";\
     else \
-      	echo "superduper/sandbox not found. Please run 'make testenv_image'";\
+      	echo "superduper/$(SUPERDUPERDB_EXTRAS) not found. Please run 'make testenv_image SUPERDUPERDB_EXTRAS=dev|sandbox'";\
       	exit -1;\
     fi
 
-	docker compose -f deploy/testenv/docker-compose.yaml up --remove-orphans &
+	MONGO_VOLUME=deploy/testenv/$(SUPERDUPERDB_EXTRAS)_data DOCKER_TAG=$(SUPERDUPERDB_EXTRAS) docker compose -f deploy/testenv/docker-compose.yaml up --remove-orphans &
```

Comment:
Want to make sure the test-environment doesn't delete the data of the sandbox experiments.